package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "mock-dapp-prototype",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "preview": "vite preview",
+    "lint": "biome lint ./src",
+    "format": "biome format ./src",
+    "check": "biome check ./src",
+    "lint:fix": "biome lint --apply ./src",
+    "format:fix": "biome format --write ./src",
+    "check:fix": "biome check --apply ./src",
+    "type-check": "bun run tsc --noEmit --skipLibCheck",
+    "pre-commit": "bun run format && bun run lint && bun run type-check"
```

checkはリンターもフォーマッターも実行してくれるので↓で良い気がする

```suggestion
    "pre-commit": "bun run check:fix && bun run type-check"
```